src/python/pants/option/global_options.py

```diff
@@ -164,12 +164,7 @@ def register_bootstrap_options(cls, register):
         default_rel_distdir = f"/{default_distdir_name}/"
 
         register(
-            "-l",
-            "--level",
-            type=LogLevel,
-            default=LogLevel.INFO,
-            help="Set the logging level. The logging levels are one of: "
-            '"error", "warn", "info", "debug", "trace".',
+            "-l", "--level", type=LogLevel, default=LogLevel.INFO, help="Set the logging level."
```

Comment:
why did we get rid of this text? printing the exact names of the valid log level values seems like very useful help information

Reply:
Repetitive:

```
 -l=<LogLevel>
  --level=<LogLevel>
  PANTS_LEVEL
  level
      one of: [trace, debug, info, warn, error]
      default: info
      current value: info
      Set the logging level. The logging levels are one of: "error", "warn", "info", "debug", "trace".
```

Reply:
What eric said: the possible values are generated automatically.